README.md

```diff
@@ -1,12 +1,29 @@
-# Aria-AT Report
-This is a web application for running manual test material from the [w3c/aria-at](https://github.com/w3c/aria-at) project, in order to evaluate the interoperability of screen reader interpretations of ARIA.
+# ARIA and Assistive Technologies App (ARIA-AT App)
+## ARIA-AT APP
+ARIA-AT aims to improve interoperability between different Assistive Technologies (ATs) in how they render ARIA patterns. This is achieved through running manual tests and presenting test results to AT vendors. The tests are based on examples from WAI-ARIA Authoring Practices, and are vetted with stakeholders following the working mode process.
+
+This app includes a reports page that shows the public results of manual testing across various browser / AT combinations. The app is also used by manual testers
+
+## About this Software
+This repo contains the react, node and graphql software for running the web application that manages testers to run the manual tests, launches the tests in an iframe, stores results, and reports results once they are reviewed.
+
+The test material and test harness themselves are managed in [w3c/aria-at](https://github.com/w3c/aria-at).
 
 ## How to run this code
-Documentation is available in the [Docs](docs) directory.
+Documentation for getting this software running locally is available in [docs/local-development.md](docs/local-development.md) directory.
+
+## Get involved
+Join the [community group](https://www.w3.org/community/aria-at/)
+Sign up as a tester
+Write more tests: https://github.com/w3c/aria-at/wiki/How-to-contribute-tests
+Review the assertions of existing test plans: https://w3c.github.io/aria-at/review-test-plans/
+Review test results
+Fix a [good first issue](https://github.com/w3c/aria-at/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```

Comment:
* Join the [community group](https://www.w3.org/community/aria-at/)
* [Sign up as a tester](https://github.com/w3c/aria-at/issues/162) for the Pilot Test (May 27 - June 2)
* [Write more tests](https://github.com/w3c/aria-at/wiki/How-to-contribute-tests)
* [Review the assertions of existing test plans](https://w3c.github.io/aria-at/review-test-plans/)
* [Review test results](https://w3c.github.io/aria-at/results/)
* Fix a [good first issue](https://github.com/w3c/aria-at/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)